src/index.js

```diff
@@ -2,14 +2,18 @@ import express from 'express';
 import trimmer from 'trim-request-body';
 import cors from 'cors';
 import debug from 'debug';
+import methodOverride from 'method-override';
 import swaggerUi from 'swagger-ui-express';
+import passport from 'passport';
+import session from 'express-session';
```

Comment:
Why are you using `express-session`, thought we are using jwt already, to make the API stateless

Reply:
Ive removed it , thanks for noting